### PR TITLE
Add some stderr output

### PR DESCRIPTION
**Pull request by pmt**
so much better...
screenshot from 2018-08-02 15-20-52

todo --
could make formatting better

### CODE REVIEW

File: internal/mill/exec.go [hunk 1]
```diff
@@ -146,6 +146,7 @@ func (p *Petsitter) start(t *skylark.Thread, fn *skylark.Builtin, args skylark.T
 	if process, err = p.Runner.StartWithStdLogs(cmdArgs, cwd, key); err != nil {
 		return nil, err
 	}
+	fmt.Printf("You started %s \n", cmdV)
```
[the remark this answers is not in the document]
Comment (nicks) [reply internal/mill/exec.go]: can you also add something like this to `run`? (there aren't currently any 'run' commands in blorgly, but i want to add some soon to setup the database)

[the remark this answers is not in the document]
Comment (pmt) [reply internal/mill/exec.go]: Yes! Thanks for the catch

File: internal/mill/exec.go [hunk 2]
```diff
@@ -324,6 +325,7 @@ func (p *Petsitter) service(t *skylark.Thread, fn *skylark.Builtin, args skylark
 		return nil, err
 	}
 
+	fmt.Printf("The service %s is now running. \n", pr.WithExposedHost(host, port).DisplayName)
```
[the remark this answers is not in the document]
Comment (pmt) [reply internal/mill/exec.go]: Thank you, thank you

**Comment by pmt** [comment]
Some things I wonder about - overlap between command output? Like, will service ever be called without start? Just wondering about which notifications are ugly/redundant

**Pull request merged**